src/algorithm/storage/storage.rs

```diff
@@ -0,0 +1,73 @@
+use serde_json::{Value, Map};
+use std::fs;
+use std::path::Path;
+use std::error::Error;
+///
+/// Класс, для хранения данных из json файла
+/// - 'file_path' - путь к файлу json
+/// - 'data' - данные json файла
+pub struct Storage {
+    file_path: String,
+    data: Value,
+}
+//
+//
+impl Storage {
+    ///
+    /// Конструктор класса
+    /// - 'file_path' - путь к файлу json
+    pub fn new(file_path: &str) -> Result<Self, Box<dyn Error>> {
```

Comment:
Should returns Self unconditionally
specify only path to folder containing files